NuGet/OpenRiaServices.Client.Core.nuspec

```diff
@@ -18,31 +18,23 @@
     <tags>WCF RIA Services RIAServices Silverlight OpenRiaServices</tags>
     <dependencies>
       <group targetFramework="net472" />
-      <group targetFramework="netstandard2.0">
-        <dependency id="System.ComponentModel.Annotations" version="5.0.0" />
-        <dependency id="System.ServiceModel.Http" version="4.10.2" />
-        <dependency id="System.ServiceModel.Primitives" version="4.10.2" />
-      </group>
-      <group targetFramework="net6.0-windows7.0">
-        <dependency id="System.ServiceModel.Http" version="6.2.0" />
-        <dependency id="System.ServiceModel.Primitives" version="6.2.0" />
-      </group>
+      <group targetFramework="net8.0" />
+	  <group targetFramework="net8.0-windows7.0" />
     </dependencies>
     <frameworkReferences>
-      <group targetFramework="net6.0-windows7.0">
+      <group targetFramework="net8.0-windows7.0">
         <frameworkReference name="Microsoft.WindowsDesktop.App.WPF" />
       </group>
     </frameworkReferences>
     <frameworkAssemblies>
       <frameworkAssembly assemblyName="System.ComponentModel.DataAnnotations" targetFramework="net472" />
       <frameworkAssembly assemblyName="System.Net.Http" targetFramework="net472" />
       <frameworkAssembly assemblyName="System.Runtime.Serialization" targetFramework="net472" />
-      <frameworkAssembly assemblyName="System.ServiceModel" targetFramework="net472" />
-      <frameworkAssembly assemblyName="System.ServiceModel.Web" targetFramework="net472" />
     </frameworkAssemblies>
   </metadata>
   <files>
     <!-- netstandard binaries -->	
+	<!--
```

Comment:
Why are we keeping so much commented code?

Reply:
I want to easily enable netstandard2.0 in a 5.6.1 release if required.